tests/checkpoint/test_checkpoint.py

```diff
@@ -941,10 +1097,10 @@ def test_newstyle_checkpoint_config_substitution_simple(
             },
         ],
         evaluation_parameters={
-            "environment": "$GE_ENVIRONMENT",
+            "environment": "my_ge_environment",
             "tolerance": 1.0e-2,
-            "aux_param_0": "$MY_PARAM",
-            "aux_param_1": "1 + $MY_PARAM",
+            "aux_param_0": "1",
+            "aux_param_1": "1 + 1",
```

Comment:
Same comment as above -- I thought that `$MY_PARAM` would work now thanks to this PR?..

Reply:
See above comment (this is in the "expected" part)